test/wrapper_array.jl

```diff
@@ -255,6 +255,17 @@ end
     @test stride(nda, :b) == 3 == stride(nda, 2) == stride(x, 2)
 end
 
+@testset "show" begin
+    nda = NamedDimsArray([1 2; 3 4], (:x, :y))
+    @test occursin("(:x, :y)", string(nda))
+    ndv = NamedDimsArray([1,2,3], :x)
+    @test occursin("], :x)", string(ndv))
+
+    str = repr(MIME"text/plain"(), nda)
+    @test occursin("NamedDimsArray(::", str)
+    @test occursin("→ :y", str)
+end
```

Comment:
these tests are only for 2-arg `show`, right? If so, would be good to add a test or two covering the other cases :)

Reply:
No, this is the 3-arg test, the `repr` form does not contain `::Matrix` (or `::Array`, depending).

Reply:
okay, cool - thanks!

would you mind add a comment saying which method each is testing? Julia's printing mechanism and what-calls-what is something that's quite hard to remember, so it's not always clear from the code (to me at least)

Reply:
At first I put more of the string, to try to make it obvious, but trimmed it because of the `Array{T,2}` vs `Matrix` printing change